ISTC.js

```diff
@@ -12,127 +12,128 @@
 	"lastUpdated": "2016-11-01 14:02:29"
```

Comment:
You have to increase the date here, in order to trigger the distribution to the clients.